src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,33 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+public class CreateDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.getRequestDispatcher("/WEB-INF/views/createDriver.jsp")
+                .forward(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+        Driver driver = new Driver();
+        driver.setName(request.getParameter("driver-name"));
+        driver.setLicenseNumber(request.getParameter("driver-license-number"));
```

Comment:
```suggestion
        driver.setName(request.getParameter("name"));
        driver.setLicenseNumber(request.getParameter("license_number"));
```
here don't need add driver, from class already understandable who's name and license number